legal-api/src/legal_api/resources/v2/namerequest.py

```diff
@@ -66,6 +60,12 @@ def validate_with_contact_info(identifier):
         orgs_response = AccountService.get_account_by_affiliated_identifier(identifier)
         if len(orgs_response['orgs']):
             return jsonify(nr_json)
+        
+        # The request must include email or phone number
+        email = request.args.get('email', None)
+        phone = request.args.get('phone', None)
+        if not (email or phone):
+            return make_response(jsonify(message='The request must include email or phone number.'), 403)
```

Comment:
To other reviewers: the changes here, which Kevin and I discussed, is to provide a response if affiliation succeeds before even checking email/phone.

This is needed because, in some cases, the UI needs to validate the NR and does not have access to the email/phone, but we can safely assume the NR is affiliated. (Eg, at start and end of a named IA.)